### DISCUSSION TITLE
some idea/suggestion that I'm not sure if possible

### DISCUSSION THREAD

**Discussion by GigglingMinister** [winstxnhdw/lc-hax#187] [Ideas]
hello! I thought that well this "hack" is very welll done so I figured out I might give some idea that might able to be implement or adapt in some way in future!
-buy commands (terminal store commands but in chat, similar to game master's)
-don't drop items upon teleport  
-kill click (smimilar to stun click but kill/delete enemy)
-ping and flash redar booster in-chat 
-disable turrets and landmine via triggermod

**Comment by winstxnhdw** [comment]
Sure, these are great ideas. Just one question.

> ping and flash redar booster in-chat

How should this be implemented? I don't really use the radar booster often.

**Reply by GigglingMinister** [reply]
it can be use to troll players who hold radar, since it does flash player and make noise
and probably use to experimenting around in single pkayer since there's no terminal guy
but yeah I didn't think this through so it's pretty weird

**Reply by winstxnhdw** [reply]
Hmm, I think such an opportunity doesn't come often so I'll probably only try and add these two.

> buy commands (terminal store commands but in chat, similar to game master's)
>don't drop items upon teleport

**Reply by winstxnhdw** [reply]
I've added the patch to prevent item drop on teleport. I'll add the buy commands on a later date. I want to get a fuzzy word matcher out first.

**Reply by GigglingMinister** [reply]
that's cool

**Reply by winstxnhdw** [reply]
Buy command is out

**Reply by GigglingMinister** [reply]
thanks
